googleapiclient/discovery.py

```diff
@@ -336,9 +361,21 @@ def build_from_document(
       authentication.
     client_options: Dictionary or google.api_core.client_options, Client options to set user
       options on the client. API endpoint should be set through client_options.
+      client_cert_source is not supported, client cert should be provided using
+      client_encrypted_cert_source instead.
+    adc_cert_path: str, client certificate file path to save the application
+      default client certificate for mTLS. This field is required if you want to
+      use the default client certificate.
+    adc_key_path: str, client encrypted private key file path to save the
+      application default client encrypted private key for mTLS. This field is
+      required if you want to use the default client certificate.
```

Comment:
No. I introduced these 2 parameters in go/python-apiary-mtls due to fact that httplib2 only reads cert/key from disk.

The following is how ADC cert/key works:

We run the cert provider command to print out the ADC cert/key to stdout, then parse the output to extract cert/key (note that now cert/key is in memory). Since httplib2 only reads cert/key from disk, they must be saved somewhere on the disk for httplib2 to load. Therefore user has to provide the cert/key file path, so we can write into these files and then load from the files.